web/ui/dashboard/src/app/private/private.component.html

```diff
@@ -23,9 +23,9 @@
 				<span class="font__weight-400 color__primary">Go to docs</span>
 			</a>
 
-			<div class="dropdown" *ngIf="organisations?.length">
```

Comment:
The issue here is that this dropdown becomes available when the user doesn't have an organisation, it'll throw an error. @Oluwadaminiola

Reply:
I moved the conditional so that the dropdown would show that there are no current organizations available and would make the `add organization` button available to the user even when they are not on the dashboard